langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiEmbeddingModel.java

```diff
@@ -80,10 +80,21 @@ public OpenAiEmbeddingModel(String baseUrl,
         this.tokenizer = getOrDefault(tokenizer, OpenAiTokenizer::new);
     }
 
+    @Override
+    protected Map<String, Integer> dimensionMap() {
+        return OpenAiEmbeddingModelName.embeddingModelDimensionMap();
+    }
+
+    @Override
     public String modelName() {
         return modelName;
     }
 
+    @Override
+    public int dimension() {
+        return dimensions;
```

Comment:
`dimensions` is an optional parameters, it can be `null`

Reply:
I remove `dimensions` in `OpenAiEmbeddingModel` because I think `dimension` in `AbstractEmbeddingModel` and `dimenions` in `OpenAiEmbeddingModel` will make user confused. So I delete it and use `dimension` in `AbstractEmbeddingModel` as an alternative way

Reply:
I meant please don't rename `dimensions` into `dimension` in `OpenAiEmbeddingModel`.

Reply:
> I meant please don't rename `dimensions` into `dimension` in `OpenAiEmbeddingModel`.

OK. I got it.

Reply:
> I meant please don't rename `dimensions` into `dimension` in `OpenAiEmbeddingModel`.

done.